loadUserData throws exception if not logged in

When loadUserData is called before the user is logged in the following exception is thrown:

```
Caused by: org.json.JSONException: Value null of type org.json.JSONObject$1 cannot be converted to JSONObject
        at org.json.JSON.typeMismatch(JSON.java:111)
        at org.json.JSONObject.<init>(JSONObject.java:160)
        at org.json.JSONObject.<init>(JSONObject.java:173)
        at org.blockstack.android.sdk.BlockstackSession$loadUserData$1.onReceiveValue(BlockstackSession.kt:104)
```

--- a/blockstack-sdk/src/main/java/org/blockstack/android/sdk/BlockstackSession.kt
+++ b/blockstack-sdk/src/main/java/org/blockstack/android/sdk/BlockstackSession.kt
@@ -99,7 +99,7 @@ class BlockstackSession(private val context: Context,
         ensureLoaded()
 
         webView.evaluateJavascript(javascript, {result ->
-                if (result != null) {
+                if (result != null && !"null".equals(result)) {
                     val newUserData = JSONObject(result)
                     userData = newUserData
                     callback(UserData(newUserData))

